docs/reference/user-management.md

```diff
@@ -0,0 +1,107 @@
+# User management
+
+User management in n8n allows you to invite people to work in your n8n instance. It includes:
+
+* Login and password management
+* Adding and removing users
+* Two account types: owner and member
+
+User management is available for self-hosted n8n. It isn't currently available for Cloud.
+
+## Setup
+
+There are three stages to set up user management in n8n:
+
+1. Configure your n8n instance to use your SMTP server.
+2. Start n8n and follow the setup steps in the app.
+3. Invite users.
+
+### Step one: SMTP
+
+You need an SMTP server for user management to send invites and password resets. Get the following information from your SMTP provider:
+
+* Server name
+* SMTP username
+* SMTP password
+* SMTP sender name
+
+To set up SMTP with n8n, configure the following environment variables for your n8n instance:
+
+| Variable | Type | Value |
+| -------- | ---- | ----- |
+| `N8N_SMTP_EMAIL_MODE` | string | smtp |
+| `N8N_SMTP_HOST` | string | _your_server_name_ |
+| `N8N_SMTP_USER` | string | _your_smtp_username_ |
+| `N8N_SMTP_PASSWORD` | string | _your_smtp_password_ |
+| `N8N_SMTP_SENDER` | string | _your_smtp_sender_name_ |
+
+If your n8n instance is already running, you need to restart it to enable the new SMTP settings.
+
+::: tip New to SMTP?
```

Comment:
What if this went first in this section?

Reply:
I'd prefer to leave it where it is. Get everyone to the key info first, then provide more for those that need it. Almost the same principle as headline -> summary -> full detail.